assets/js/renderer.js

```diff
@@ -370,11 +370,11 @@ $(document).ready(function () {
             });
     }
 
-    // Open a SQLite Database File
-    openDBFileButton.click(function() {
+    // Open a SQLite Database File -- refactored to work with both button click and custom menu
+    const openSQLite = () => {
```

Comment:
Can you change the fat arrow to a regular function?

`const openSQLite = () => { ... }`

Change to:

`function openSQLite() { ... }`